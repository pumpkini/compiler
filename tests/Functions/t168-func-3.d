
void test(int x, int y, int z) {
    Print("args: ", x, " ", y, " ", z);
}

int main() {
    test(ReadInteger(), ReadInteger(), ReadInteger());
}
