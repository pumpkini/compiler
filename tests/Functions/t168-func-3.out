args: 8 5 4
